### ISSUE TITLE
Pending QUnit.todo tests should be listed with "hide passed" checked.

### ISSUE THREAD

**Issue by rwjblue** [qunitjs/qunit#1125]
I started experimenting with the new (to me at least) `QUnit.todo` feature that exists in qunitjs@2.2.1, and I really like it! Thank you all for working so hard to implement it.

I noticed a few things that stuck out to me while using todo's, and I would like to submit a PR to fix but would like guidance on the best approach before beginning.

In the default HTML reporter tests ran as `QUnit.todo`'s get a visual indication to the left of the test, saying `[todo]`:

![todo screenshot](https://monosnap.com/file/Ge11ij3j1HPqTVA7bi7CzE3zur1gor.png)

The mental model that I have when using `QUnit.todo` is that I am quickly putting down tests (often TDD where the test is written before the implementation even exists), and then once I have the tests written I am focusing on making them pass.  In a large test suite, these `[todo]` entries are difficult to find when skimming the list. Unlike skipped tests (which are also technically not failures) which are still displayed when "Hide Passed" is checked, these todo tests do not show up when the "Hide Passed" checkbox is checked.

**tldr;** I would like to have `QUnit.todo` tests not be hidden by "Hide Passed", and instead display them (similar to skipped tests) with an alternate color or other visual treatment in the test list.

**Comment by platinumazure** [comment]
I would be :+1: to this change. Let's see if the rest of @qunitjs/qunit-team agrees.

**Comment by trentmwillis** [comment]
I am definitely in favor of this. The change itself should be relatively easy, just need to adjust some CSS [here](https://github.com/qunitjs/qunit/blob/41cb62569786de4b0d64c48cd0bf099839e28ae4/src/qunit.css#L234-L247) I believe.

**Comment by Krinkle** [comment]
LGTM. Consistency with skipped tests, and furthers the model of working from "Hide Passed" as a checklist for what it left to do. (Be it failed, skipped, or todo tests.)

**Comment by leobalter** [comment]
I'm 👍  to this while we can also go further and add individual filters for each status in a future. 

It would be nice to also check only skipped or todo tests.
